deploy-manage/deploy/cloud-on-k8s/required-rbac-permissions.md

```diff
@@ -1,8 +1,12 @@
 ---
+applies:
+  eck: all
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-eck-permissions.html
 ---
 
+% candidate for REFERENCE content.
```

Comment:
I like this here personally - they're kind of like prerequisites so keeping them close to the instructions is nice

Reply:
perfect.